lib/shared/errors.js

```diff
@@ -123,29 +123,6 @@ const getUserFriendlyMessageProperty = (error, property) => {
 };
 
 /**
- * @summary Check whether an error should be reported to TrackJS
- * @function
- * @public
- *
- * @description
- * In order to determine whether the error should be reported, we
- * check a property called `report`. For backwards compatibility, and
- * to properly handle errors that we don't control, an error without
- * this property is reported automatically.
- *
- * @param {Error} error - error
- * @returns {Boolean} whether the error should be reported
- *
- * @example
- * if (errors.shouldReport(new Error('foo'))) {
- *   console.log('We should report this error');
- * }
- */
-exports.shouldReport = (error) => {
```

Comment:
Yeah, all that logic has been moved to `resin-corvus`, so we can re-use it with other resin.io products.